src/clients/search/index.ts

```diff
@@ -0,0 +1,58 @@
+import { IClaim } from '../../models/claim';
+import { ITargetParams } from '../target';
+
+export interface ISearchParams extends ITargetParams {
+  query?: string;
+}
+export interface ISearchClient {
+  search: (params: ISearchParams) => Promise<IClaim | Error>;
+}
+
+/**
+ * Cliuent class that cxommunicates with the cse api
+ * @class {SearchClient}
+ */
+export class SearchClient implements ISearchClient {
```

Comment:
this code must be from branching off of my earlier branch, I would say that it is not ready to be merged yet but i know that there are 6 other PRs active and some of the other developers have been refactoring this code. considering the time frame we're working with, I think we should merge this in.